src/main/scala/beam/agentsim/scheduler/BeamAgentScheduler.scala

```diff
@@ -242,6 +245,9 @@ class BeamAgentScheduler(
     case Terminated(actor) =>
       terminateActor(actor)
 
+    case RequestCurrentTime =>
```

Comment:
I believe this was added by me and was meant to be a debugging thing. It should be removable - anybody disagree?

Reply:
nope - done